tests/integration_tests/test_game.py

```diff
@@ -532,3 +537,241 @@ async def test_gamestate_ended_clears_references(
 
     assert test.lobby_connection.game_connection is None
     assert rhiza.lobby_connection.game_connection is None
+
+
+@pytest.mark.rabbitmq
+@fast_forward(30)
+async def test_galactic_war_1v1_game_ended_broadcasts_army_results(lobby_server, channel):
+    mq_proto_all = await connect_mq_consumer(
+        lobby_server,
+        channel,
+        "success.gameResults.create"
+    )
+
+    host_id, _, host_proto = await connect_and_sign_in(
+        ("test", "test_password"), lobby_server
+    )
+    guest_id, _, guest_proto = await connect_and_sign_in(
+        ("Rhiza", "puff_the_magic_dragon"), lobby_server
+    )
+    await read_until_command(guest_proto, "game_info")
+
+    # Set up the game
+    game_id = await host_game(host_proto, mod="gw")
+    await join_game(guest_proto, game_id)
+    # Set player options
+    await send_player_options(
+        host_proto,
+        [host_id, "Army", 1],
+        [host_id, "Team", 1],
+        [host_id, "StartSpot", 0],
+        [host_id, "Faction", 1],
+        [host_id, "Color", 1],
+        [guest_id, "Army", 2],
+        [guest_id, "Team", 2],
+        [guest_id, "StartSpot", 1],
+        [guest_id, "Faction", 2],
+        [guest_id, "Color", 2],
+    )
+
+    # Launch game
+    await host_proto.send_message({
+        "target": "game",
+        "command": "GameState",
+        "args": ["Launching"]
+    })
+
+    await read_until(
+        host_proto,
+        lambda cmd: cmd["command"] == "game_info" and cmd["launched_at"]
+    )
+    await host_proto.send_message({
+        "target": "game",
+        "command": "EnforceRating",
+        "args": []
+    })
+
+    # End the game
+    # Report results
+    for proto in (host_proto, guest_proto):
+        await proto.send_message({
+            "target": "game",
+            "command": "GameResult",
+            "args": [1, "victory 10"]
+        })
+        await proto.send_message({
+            "target": "game",
+            "command": "GameResult",
+            "args": [2, "recall defeat -5"]
+        })
+    # Report GameEnded
+    for proto in (host_proto, guest_proto):
+        await proto.send_message({
+            "target": "game",
+            "command": "GameEnded",
+            "args": []
+        })
+    # Now disconnect both players
+    for proto in (host_proto, guest_proto):
+        await proto.send_message({
+            "target": "game",
+            "command": "GameState",
+            "args": ["Ended"]
+        })
+
+    message = await asyncio.wait_for(mq_proto_all.read_message(), timeout=5)
+
+    assert message == {
+        "game_id": 41956,
+        "rating_type": "global",
+        "map_id": 7,
+        "featured_mod": "gw",
+        "sim_mod_ids": [],
+        "commander_kills": {},
+        "validity": "VALID",
+        "teams": [
+            {
+                "outcome": "DEFEAT",
+                "player_ids": [3],
+                "army_results": [[3, 2, "DEFEAT", ["recall"]]]
```

Comment:
I've been thinking maybe it would be better to have these be a dictionary than a list.
```suggestion
                "army_results": [
                    {
                        "player_id": 3, 
                        "team": 2, 
                        "outcome": "DEFEAT", 
                        "metadata": ["recall"]
                    }
                ]
```

Reply:
That's a good idea 👍

Reply:
Is it weird to you that in my 2v1 scenario there are three items in `message['teams']`? I'd have expected two of those to be combined into a single entry?

Reply:
Looks like your team setup isn’t quite right.